test/spec/test-rule-spec.js

```diff
@@ -85,6 +104,16 @@ function fakeCheckRuleWithReports() {
   return { check };
 }
 
+function fakeCheckRuleWithPath() {
+  function check(node, reporter) {
+    if (is(node, 'Definitions')) {
+      reporter.report(node.id, 'Definitions detected', [ 'extensionElements', 'values', 0 ]);
```

Comment:
Should I change for a better example here?

Reply:
I'd probably have chosen something like `[ 'foo', 'bar', 'baz' ]` so this already is a better example in the sense of one that makes sense. 🤣